src/videotoolbox.cs

```diff
@@ -504,6 +530,14 @@ interface VTVideoEncoderSpecificationKeys {
 
 		[Field ("kVTVideoEncoderSpecification_EncoderID")]
 		NSString EncoderID { get; }
+		
+		[Mac (10,14)]
+		[Field ("kVTVideoEncoderSpecification_RequiredEncoderGPURegistryID")]
+		NSString RequiredEncoderGPURegistryID { get; }
```

Comment:
`RequiredEncoderGPURegistryID` => `RequiredEncoderGPURegistryId` (yep we missed the above one`